AstroCloud CLI Reference Guide (#294)
astronomer/docs: cloud/cli-reference/astrocloud-workspace-switch.md

---
sidebar_label: "astrocloud workspace switch"
title: "astrocloud workspace switch"
id: astrocloud-workspace-switch
description: Reference documentation for astrocloud workspace switch.
---

## Description

> Switch to another Workspace. After running the command, you are presented with a list of all Workspaces that you have access to. Then, to switch to a given Workspace, enter the number next to the Workspace's name in the list.

Reviewer: ```suggestion
Switch to another Workspace. After running the command, you will be prompted to select from a list of all Workspaces in your Organization that you have access to. To switch to a given Workspace, enter the number next to the Workspace's name in the list.
```

## Usage

```sh
astrocloud workspace switch
```

## Related Commands

> - [`astrocloud workspace switch`](cli-reference/astrocloud-workspace-list.md)

Reviewer: ```suggestion
- [`astrocloud workspace list`](cli-reference/astrocloud-workspace-list.md)
```
